docs/vizier/vizier.rst

```diff
@@ -26,15 +26,13 @@ the :meth:`~astroquery.vizier.VizierClass.find_catalogs` tool can be used:
 
     >>> from astroquery.vizier import Vizier
     >>> catalog_list = Vizier.find_catalogs('Kang W51')
-    >>> print(catalog_list)
-    {u'J/ApJ/706/83': <astropy.io.votable.tree.Resource at 0x108d4d490>,
-     u'J/ApJS/191/232': <astropy.io.votable.tree.Resource at 0x108d50490>}
-    >>> print({k:v.description for k,v in catalog_list.items()})
-    {u'J/ApJ/706/83': u'Embedded YSO candidates in W51 (Kang+, 2009)',
-     u'J/ApJS/191/232': u'CO survey of W51 molecular cloud (Bieging+, 2010)'}
+    >>> print(['{!r}: {!r}'.format(k, v.description)
+    ...     for k,v in catalog_list.items()])
+    ["'J/ApJ/706/83': 'Embedded YSO candidates in W51 (Kang+, 2009)'",
+     "'J/ApJS/191/232': 'CO survey of W51 molecular cloud (Bieging+, 2010)'"]
```

Comment:
I think the printed dict view is a little nicer, since it avoids the quotes-around-quotes.  Does `{!r}` mean "call the `__repr__`"?